doc/design/modules/neural-network.tex

```diff
@@ -117,6 +117,24 @@ \subsubsection{Backpropagation}
 \[\boxed{\delta_{k}^j = \sum_{t=1}^{n_{k+1}} \left( \delta_{k+1}^t \cdot u_{k}^{jt} \right) \cdot \phi'(\mathit{net}_{k}^j)}\]
```

Comment:
Please add your name to the list of authors in the beginning of the file. Maybe add the history for this file as well?